### ISSUE TITLE
Error while importing PDF

### ISSUE THREAD

**Issue by lkjts** [mike-edel/ID-MultiPageImporter#15]
Trying to import a PDF always results with this error:

```
"***.pdf" doesn't contain a "All Layers Bounding Box" crop type: Please try again by selecting a different crop type or open
the PDF in Acrobat and perform a "Save As..." command.

```
Than this:

`JavaScript Error 89867 The default engine 'main' cannot be deleted. Line: 589`

I think it has something to do with Crop margins not being defined in the PDF. Since I'm stuck with this particular PDF I'm hoping for a solution.

**Comment by mike-edel** [comment]
Sounds like you will have to fix the PDF first. Acrobat DC (pro) will allow you to set manual crop regions. That should do it.
What happens when you place the pdf manually?

**Comment by lkjts** [comment]
It turns out the problem was indeed in the PDF. The PDF Was created from Microsoft Word on a Mac, the same Mac I used to import it into InDesign. After some trial and error the only thing that worked was creating the same PDF from the original Word file but on a PC. Then I used the new PDF on the same Mac to import into InDesign and it worked just fine.

Side note, I did use Acrobat on the first PDF to set the bounding boxes but it did not help.
